setup.py

```diff
@@ -41,6 +41,7 @@
     'google-auth>=1.4.1',
     'google-auth-httplib2>=0.0.3',
     'Jinja2>=2.9.5',
+    'jmespath',
```

Comment:
Does version matter?

Reply:
It's probably better to be conservative.  I'll add a min version requirement on the current release.